.bazelversion

```diff
@@ -0,0 +1,2 @@
+6.4.0
+# NOTE: Bazel 7.0 errors out on 'buildifier_lint_check' with a hint to 'go_register_toolchains' in 'setup.bzl'
```

Comment:
@dkroenke maybe you have more luck than me but I was not able to get the linter working with the latest bazel release from yesterday.